include/flecs/addons/meta.h

```diff
@@ -202,6 +202,7 @@ typedef struct EcsMember {
     int32_t count;                                 /**< Number of elements (for inline arrays). */
     ecs_entity_t unit;                             /**< Member unit. */
     int32_t offset;                                /**< Member offset. */
+    bool explicit_offset;                          /**< if Offset should be explicitly used. */
```

Comment:
Should probably also get the capitalization fix.

Reply:
yup, noticed when the CI failed haha :D, thanks !